### DISCUSSION TITLE
How to know if an existing sqlproject is SDK-styled?

### DISCUSSION THREAD

**Discussion by abceleung** [microsoft/DacFx#382] [Q&A] [answered]
Hi, I have a sqlproject created with Azure Data Studio. How to know if it is a SDK-styled sqlproject?

**Comment by dzsquared** [comment, marked as answer]
The quickest way to see if a project is an SDK-style project is to open the .sqlproj file.  In Azure Data Studio there's a menu item on the project node in the projects pane for "Edit .sqlproj File" as well as the file explorer view.

If the top of the project looks similar to:
```xml
<?xml version="1.0" encoding="utf-8"?>
<Project DefaultTargets="Build">
  <Sdk Name="Microsoft.Build.Sql" Version="0.1.12-preview" />
```
then it is an SDK-style project.  The key here is that element `<SDK Name="Microsoft.Build.Sql" Version="0.1.x-preview" />`

Otherwise, it's an original style project.
